client/rpc/tx.go

```diff
@@ -102,10 +102,10 @@ func WaitTxCmd() *cobra.Command {
 		Short:   "Wait for a transaction to be included in a block",
 		Long:    `Subscribes to a CometBFT WebSocket connection and waits for a transaction event with the given hash.`,
 		Example: fmt.Sprintf(`By providing the transaction hash:
-$ %[1]sd q wait-tx [hash]
+$ %[1]s q wait-tx [hash]
```

Comment:
nit: I don't think this is correct.

Reply:
The `%s` is filled with `version.AppName` which is supposed to be the name of the executable. Adding an extra `d` to the end then makes the example incorrect.

E.g. In Provenance's stuff, `version.AppName` is `provenanced`. Without this change, the examples end up looking like `$ provenancedd q wait-tx [hash]`, which isn't correct.